scapy/as_resolvers.py

```diff
@@ -13,29 +13,40 @@
 from scapy.config import conf
 from scapy.compat import plain_str
 
+from scapy.compat import (
+    Any,
+    Optional,
+    Tuple,
+    List,
+)
+
 
 class AS_resolver:
     server = None
-    options = "-k"
+    options = "-k"  # type: Optional[str]
 
     def __init__(self, server=None, port=43, options=None):
+        # type: (Optional[str], int, Optional[str]) -> None
```

Comment:
s/int/Optional[int]/ ?

Reply:
`Optional` is an alias for `Union[..., None]`

Reply:
Thanks!